examples/meta/generator/targets/java.json

```diff
@@ -107,14 +109,14 @@
         "Access": {
             "Vector": "$identifier.get($indices)",
             "Matrix": "$identifier.get($indices)",
-            "BoolVector": "$identifier.get_element($indices)",
-            "BoolMatrix": "$identifier.get_element($indices)"
+            "BoolVector": "$identifier.get($indices) == 1.0 ? true : false",
+            "BoolMatrix": "$identifier.get($indices) == 1.0 ? true : false"
         },
         "Assign": {
             "Vector": "$identifier.put($indices, $expr)",
             "Matrix": "$identifier.put($indices, $expr)",
-            "BoolVector": "$identifier.set_element($expr, $indices)",
-            "BoolMatrix": "$identifier.set_element($expr, $indices)"
+            "BoolVector": "$identifier.put($indices, $expr ? 1.0 : 0.0)",
+            "BoolMatrix": "$identifier.put($indices, $expr ? 1.0 : 0.0)"
```

Comment:
This is also not great, but at least we consistently use DoubleMatrix, rather than mix with a shogun type